api/src/db-connection.ts

```diff
@@ -257,9 +261,15 @@ export class Connection {
         ...relationships,
       },
     };
+
+    if (isSymbol(entity)) {
+      entity.attributes.disambiguated_id = entity.attributes.mathml
```

Comment:
nit: `;` (sorry this should really have prettier set up to handle this already...)

Reply:
😢 

I've been out of the JS game for a while, can you tell?